src/components/list/list.js

```diff
@@ -244,6 +244,7 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
           wrapIn('div');
         } else if (!tEl[0].querySelector('md-button:not(.md-secondary):not(.md-exclude)')) {
           tEl.addClass('md-no-proxy');
+          tEl.addClass('md-list-item-content');
```

Comment:
Are you sure this class should be added during the compile phase?

if so i'd minify the calls to:
```js
tEl.addClass('md-no-proxy md-list-item-content');
```